maywood/sass/style-child-theme.scss

```diff
@@ -5,7 +5,7 @@ Author: Automattic
 Author URI: https://automattic.com/
 Description: Maywood is a refined theme designed for restaurants and food-related businesses seeking a modern look.
 Requires at least: WordPress 4.9.6
-Version: 1.1.4
+Version: 1.2.4
```

Comment:
I'm not an expert in semver, but don't we need to reset the patch version when we increment the minor version? ie `1.2.0` instead of `1.2.4`?

Reply:
fixed at 8e2e524